pkg/pillar/devicenetwork/dns_test.go

```diff
@@ -161,7 +162,7 @@ func TestDnsResolveTimeout(t *testing.T) {
 }
 
 func TestResolveWithPortsLambda(t *testing.T) {
-	t.Parallel()
+	defer goleak.VerifyNone(t, goleak.IgnoreAnyFunction("go.opencensus.io/stats/view.(*worker).start"), goleak.IgnoreCurrent())
```

Comment:
:clap: :clap: :clap: 
leak sanitizer is a good thing, can we enable it test-wide?

Reply:
You mean other tests?
I fear we will find more leaks and also more false-positives (like this opencensus thing).

Also it cannot run with `t.Parallel()` and needs a `TestMain(...)`.

Reply:
> I fear we will find more leaks

😅😅😅
I think that's exactly what we need: find the leaks before users face the consequences of them =) But not within the scope of this PR.